docker-compose.yml

```diff
@@ -9,6 +9,10 @@ services:
       - 8000:8000
     environment:
       - PORT=8000
+    volumes:
```

Comment:
@joshwyatt could you please comment on if this the right way to do this? thanks

Reply:
@SSabu Yeah these changes are in fact present in the dev branch (the branch you're talking to merge into with this PR. I'm not sure why it's showing as a change from you here. Perhaps you could rebase from upstream dev and see that fixes it. thanks.